src/components/molecules/Footer/footer.scss

```diff
@@ -0,0 +1,32 @@
+.page-footer {
+	a {
+		color: #6c757d !important;
+	}
+	a:hover {
+		color: lighten(#6c757d, 10%) !important;
+	}
+
+	li {
+    color: white;
```

Comment:
This indentation seems a bit messed up to me

Reply:
Only on this GitHub view. Something weird with tabs and spaces.
Not an issue for now.